lisa/tools/fio.py

```diff
@@ -0,0 +1,82 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+from typing import TYPE_CHECKING, List, Type, cast
+
+from lisa.executable import Tool
+from lisa.operating_system import Posix
+
+from .git import Git
+
+if TYPE_CHECKING:
+    from .make import Make
+
+
+class Fio(Tool):
+    fio_repo = "https://github.com/axboe/fio/"
+    branch = "fio-3.28"
+
+    @property
+    def command(self) -> str:
+        return "fio"
+
+    @property
+    def can_install(self) -> bool:
+        return True
+
+    @property
+    def dependencies(self) -> List[Type[Tool]]:
+        return [Git, Make]
+
+    def _install_from_src(self) -> bool:
+        tool_path = self.get_tool_path()
+        self.node.shell.mkdir(tool_path, exist_ok=True)
+        git = self.node.tools[Git]
+        git.clone(self.fio_repo, tool_path)
+        code_path = tool_path.joinpath("fio")
+        git.checkout(
+            ref="refs/heads/master", cwd=code_path, checkout_branch=self.branch
+        )
+        make = self.node.tools[Make]
+        make.make_install(cwd=code_path)
+        return self._check_exists()
+
+    def install(self) -> bool:
+        posix_os: Posix = cast(Posix, self.node.os)
+        posix_os.install_packages("fio")
+        if not self._check_exists():
+            self._install_from_src()
+        return self._check_exists()
+
+    def launch(
+        self,
+        name: str,
+        filename: str,
+        mode: str,
+        runtime: int,
```

Comment:
1. Give a default value
2. How about 10 seconds?
3. The name is confusing, "time" may be enough.

Reply:
updated the name, 10 seconds is too short, the fio need warm up.

Reply:
If 10 seconds is too short, please give another longer default value. The 120 is too long, we can try 20, 30, 60 first.

Reply:
currently use 60 s